templates/10-Operational Analysis/operational-capability.html.j2

```diff
@@ -53,64 +64,40 @@
 {# Q: should we resolve inherited activities and actors / explicitly indicate where an actor / activity got inherited #}
 {{ object.context_diagram }}
 
-<h2>Involved entities and activities</h2>
-To enable the Capability, the following entities and activities are involved:
-{%- set entities = object.involved_entities | list + object.involved_activities | map(attribute="owner") | unique(attribute="uuid") | list -%}
-{%- if entities -%}
+{{
+  render_data_flow_and_involved_activities(
+    object,
+    "Involved Entities and Activities",
+    object.involved_activities,
+    object.involved_entities,
+    "The diagram below provides an overview of the operational activities that
+    are required to realize the Capability.",
+    "To enable the Capability, the following entities and activities are
+    involved:",
+    "operational Capability",
+    "activities",
+    "entities",
+    "Entity"
+  )
```

Comment:
That's a single macro call. What the actual fuck. You can't tell me that this is more maintainable than copying the macro contents into each layer template.

Aside from some very minor differences (mostly the layer name, which can be inferred from `object.layer`, and some that are probably just copy-paste mistakes), all Capability templates except for OperationalCapability are exactly identical. Let's combine them into a Capability master template, and use `{% extends %}` and `{% block %}` directives instead of unwieldy huge macro calls with a bunch of unnamed positional arguments.

Depending on how that goes and how many blocks we'd need, we can then decide whether it makes more sense to copy the template or reuse it for OA as well.